### PR TITLE
student class fix

### PR DESCRIPTION
**Pull request by LeonardoSoto34**
I saw that u put parenthesis to fix the class header

### CODE REVIEW

File: src/main/java/com/csase/Student.java [hunk 1]
```diff
@@ -13,7 +13,7 @@ public class Student {
     /*
      * Sets study planner and scanner
      */
-    public Student {
+    public Student() {
```
Comment (sprshv) [inline src/main/java/com/csase/Student.java:16]: added parentheses to make it a method

**Review by sprshv — COMMENTED** [review]
very nice changes

**Comment by LeonardoSoto34** [comment]
thank you for fixing the class header.

File: src/main/java/com/csase/Student.java (hunk 1 above)
Comment (codingwes) [reply src/main/java/com/csase/Student.java:16]: Don't forget to remove the duplicate line 17

**Pull request merged**